addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -132,7 +132,11 @@ options.registry.blog_cover = options.Class.extend({
         this.$image.css("background-image", "");
     },
     change: function (previewMode, value, $li) {
-        var $image = $("<img/>", {src: this.$image.css("background-image")});
+        var $image = $("<img/>");
+        var background = this.$image.css("background-image");
+        if (background && background !== "none") {
+            $image.attr('src', background.match(/url\(["']?(.*)["']?\)/)[1]);
```

Comment:
- You should add `^`/`$` to mark beginning/ending
- `(.*)` will catch the last `'` or `"`, so your last `["']?` will never match anything